src/main/java/com/google/cloud/resourcemanager/Lien.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.resourcemanager.ResourceManager.LienListOption;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.util.List;
+import java.util.Objects;
+
+public class Lien extends LienInfo {
+  private static final long serialVersionUID = 6767630161335155133L;
+
+  private final ResourceManagerOptions options;
+  private transient ResourceManager resourceManager;
+
+  /** Builder for {@code Lien}. */
+  public static class Builder extends LienInfo.Builder {
+    private final ResourceManager resourceManager;
+    private final LienInfo.BuilderImpl infoBuilder;
+
+    Builder(Lien lien) {
+      this.resourceManager = lien.resourceManager;
+      this.infoBuilder = new LienInfo.BuilderImpl(lien);
+    }
+
+    @Override
+    public Builder setCreateTime(String createTime) {
+      infoBuilder.setCreateTime(createTime);
+      return this;
+    }
+
+    @Override
+    public Builder setName(String name) {
+      infoBuilder.setName(name);
+      return this;
+    }
+
+    @Override
+    public Builder setOrigin(String origin) {
+      infoBuilder.setOrigin(origin);
+      return this;
+    }
+
+    @Override
+    public Builder setParent(String parent) {
+      infoBuilder.setParent(parent);
+      return this;
+    }
+
+    @Override
+    public Builder setReason(String reason) {
+      infoBuilder.setReason(reason);
+      return this;
+    }
+
+    @Override
+    public Builder setRestrictions(List<String> restrictions) {
+      infoBuilder.setRestrictions(restrictions);
+      return this;
+    }
+
+    @Override
+    public Lien build() {
+      return new Lien(resourceManager, infoBuilder);
+    }
+  }
+
+  Lien(ResourceManager resourceManager, LienInfo.BuilderImpl infoBuilder) {
+    super(infoBuilder);
+    this.resourceManager = checkNotNull(resourceManager);
+    this.options = resourceManager.getOptions();
+  }
+
+  /** Returns the {@link ResourceManager} service object associated with this Lien. */
+  public ResourceManager getResourceManager() {
+    return resourceManager;
+  }
+
+  /**
+   * Create a Lien which applies to the resource denoted by the parent field.
+   *
```

Comment:
Create -> Creates

Reply:
it will create only one `lien`  so no need to make it `creates` same thing for all.

Reply:
It's a requirement for javadoc comment:

https://developers.google.com/style/reference-verbs